app/build.gradle

```diff
@@ -307,6 +307,7 @@ dependencies {
     because "Traceur is incompatible with the version of RxJava this artifact pulls in transitively"
   }
   testImplementation "org.threeten:threetenbp:$versions.threeTenBp"
+  implementation "com.github.f4b6a3:uuid-creator:1.4.1"
```

Comment:
Would you consider using a library from a more reliable source?

https://github.com/cowtowncoder/java-uuid-generator

This library is from the Jackson folks.

Reply:
The library we are using right now is already tested with between app and server.
If you think we should still update, I will make this change.